sql/transactions.js

```diff
@@ -44,23 +44,21 @@ var TransactionsSql = {
 		].filter(Boolean).join(' ');
 	},
 
-	getById: 'SELECT *, ENCODE ("t_senderPublicKey", \'hex\') AS "t_senderPublicKey", ENCODE ("m_recipientPublicKey", \'hex\') AS "m_recipientPublicKey" FROM trs_list WHERE "t_id" = ${id}',
+	getTransferByIds: 'SELECT "transactionId" as "transaction_id", CONVERT_FROM(data, \'utf8\') AS "tf_data" FROM transfer WHERE "transactionId" IN (${id:csv})',
```

Comment:
Why are these snakecase ?

Reply:
our transaction `dbRead` function expect transaction column values in snake cases. So, I following existing approach.